beard/clustering/blocking.py

```diff
@@ -97,58 +151,41 @@ def _validate(self, X, blocks):
 
         return X, blocks
 
-    def _fit(self, X, y, blocks):
+    def _fit(self, X, y, blocks, n_jobs):
         """Fit base clustering estimators on X."""
+        def _grouper(size, seq):
+            return (seq[pos:pos + size] for pos in xrange(0, len(seq), size))
```

Comment:
You should not use `xrange`. It does not save much and it is not Python3 compatible.